app/src/pages/inside/membersPage/membersPage.jsx

```diff
@@ -99,6 +112,28 @@ export class MembersPage extends Component {
 
   getBreadcrumbs = () => [{ title: this.props.intl.formatMessage(messages.membersPageTitle) }];
 
+  getNoItemMessage = () => {
+    let output = '';
+    if (this.props.filter !== null && this.props.filter !== '') {
+      output = (
+        <div className={cx('filter-not-found')}>
+          <p className={cx('filter-not-found-text')}>
+            <i className={cx('filter-not-found-icon')}>{Parser(ErrorIcon)}</i>
+            {Parser(
+              this.props.intl.formatMessage(messages.membersNotFound, {
+                filter: `<span>${this.props.filter}</span>`,
+              }),
+            )}
+          </p>
+          <p className={cx('filter-not-found-hint')}>
+            {this.props.intl.formatMessage(messages.checkQuery)}
+          </p>
+        </div>
+      );
+    }
+    return output;
```

Comment:
Is some cases this function returns empty string or node? I guess check `(this.props.filter !== null && this.props.filter !== '')` should be somewhere outside and method should always return node.